lib/Settings.js

```diff
@@ -43,6 +45,9 @@ Settings.loadRepositoriesSettings = function (configPath) {
         if (!defined(configJson[name]['gitHubToken'])) {
             return Promise.reject('repository ' + name + ' must have a `gitHubToken`');
         }
+        configJson[name]['checkChangesMd'] = parseBoolean(configJson[name]['checkChangesMd']);
```

Comment:
We want both of these settings to default to true, but the way parseBoolean works, these will default to false.